bin/eshost.js

```diff
@@ -45,6 +45,9 @@ const yargv = yargs
   .describe('showSource', 'show input source')
   .boolean('showSource')
   .alias('showSource', 'i')
+  .describe('quorum', 'If all engines agree, exit(0) with no output, otherwise print and exit(1); implies --coalesce')
+  .boolean('quorum')
+  .alias('quorum', 'q')
```

Comment:
Yes, good point. That's more accurate to the intent anyway. Short form `-u` is also available.